node/utils/allowListAccounts.ts

```diff
@@ -0,0 +1,21 @@
+const appName = process.env.VTEX_APP_ID as string
+
+interface StoreGraphQLSettings {
+  trustedAccounts?: string[]
+}
+
+export const allowedAccounts = async (
+  context: Context,
+  tokenAccount: string
+) => {
+  // check if account exists in allow list and the user token account exists for that account.
```

Comment:
I'm a little confused with this comment. What is being checked in this snippet is really if the account is in the allowed list or if the allowed list exists?
```suggestion
  // check that there is an allowed list for the current account and that the list includes the user token account.
```
Here is a suggestion, but I'm not sure that the comment is necessary